xinference/client.py

```diff
@@ -58,10 +58,42 @@ def generate(
             Union["LlamaCppGenerateConfig", "PytorchGenerateConfig"]
         ] = None,
     ) -> Union["Completion", Iterator["CompletionChunk"]]:
+        """
+        Function to call the generate functionality of model on the server side.
```

Comment:
Creates a completion for the provided prompt and parameters.

Reply:
resolved in the latest version